pandas/core/frame.py

```diff
@@ -7441,11 +7443,12 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
         Specifying `numeric_only=False` will also compute the quantile of
         datetime and timedelta data.
 
-        >>> df = pd.DataFrame({'A': [1, 2],
-                               'B': [pd.Timestamp('2010'),
-                                     pd.Timestamp('2011')],
-                               'C': [pd.Timedelta('1 days'),
-                                     pd.Timedelta('2 days')]})
+        >>> df = pd.DataFrame({ 'A': [1, 2], \
+                                'B': [pd.Timestamp('2010'), \
+                                      pd.Timestamp('2011')], \
+                                'C': [pd.Timedelta('1 days'), \
+                                      pd.Timedelta('2 days')]})
```

Comment:
If possible, try to define just an example at the beginning, and use it everywhere. If you need a date column, you can add it there.

Reply:
okay, @datapythonista .

Reply:
okay, @datapythonista .